packages/app/src/app/overmind/namespaces/dashboard/actions.ts

```diff
@@ -2197,6 +2197,19 @@ export const removeRepositoryFromTeam = async (
 
   try {
     if (assignedTeam?.id) {
+      const confirmed = await actions.modals.alertModal.open({
+        title: 'Are you sure?',
```

Comment:
Should the title be "Remove repository" with the text "Are you sure you want to remove the repository? This will remove the repository and all the branches form CodeSandbox. Any code that is not pushed on GitHub will be lost". Feels like the title gives more context about the action that way, not sure if there are UX guidelines for this we want to follow?